simulators/README.md

```diff
@@ -52,24 +100,57 @@ Detailed feature list:
+    where "type" is optional and its default value is the value from the "fragment" property.
+  
+  - In measurements, `valueDistribution` is defined to let the simulator know which distribution formular to use to generate measurements. There are three choices that can be defined here: `uniform`, `uniformint`, `normal`.
+
+- Simulates shutdowns (no events or measurements are sent if simulator is DOWN or out of shift)
+
 - the main entry point is [simulator.py](main/simulator.py)
   - the script reads the configuration from [simulator.json](main/simulator.json) and creates a new device for every entry
   - the `id` property is used as `external_id` for the ManagedObjects to avoid creating multiple devices when redeploying/updating the microservice
+  
 - Simulators act according to given Shiftplans
   - Simulators are linked to shiftplans via locationId
-  - Shiftplans are polled once a day
-  - If a Simulator is not in Production time according to the given Shiftplan, it will not produce any  events
-  - At startup [shiftplans.json](main/shiftplans.json) is parsed and used to update the given timeslots whitin the shiftplan
-  - The `locationId's` presented in [shiftplans.json](main/shiftplans.json) are used to parse for all shiftplans used in the script and are the only shiftplans are considered for the polling
+  - If a Simulator is not in Production time according to the given Shiftplan, it will not produce any events and measurements.
+  - At startup [shiftplans.json](main/shiftplans.json) is parsed and used to update the given timeslots within the shiftplan.
+  - The `locationId's` presented in [shiftplans.json](main/shiftplans.json) are used to parse for all shiftplans used in the script but only a specific shiftplan is applied to a simulator if it define the `locationId` of a shift.
+  ```
+    {
+      "type": "Simulator",
+      "id": "sim_012",
+      "label": "Normal #12 with short shifts",
+      "locationId": "ShortShiftsLocation",
+      "enabled": true
+    }
+  ```
+  In this example, the ShortShiftsLocation shift is applied to the simulator sim_012.
+
+### Shiftplan definition
+```
+  [
+    {
+      "locationId": "OneShiftLocation",
+      "recurringTimeslots": [
+        { "id": "OneShiftLocation-DayShift", "seriesPostfix": "DayShift", "slotType": "PRODUCTION", "slotStart": "2022-07-13T08:00:00Z", "slotEnd": "2022-07-13T16:00:00Z", "description": "Day Shift", "active": true, "slotRecurrence": { "weekdays": [1, 2, 3, 4, 5] } },
+        { "id": "OneShiftLocation-Break", "slotType": "BREAK", "slotStart": "2022-07-13T12:00:00Z", "slotEnd": "2022-07-13T12:30:00Z", "description": "Day Shift Break", "active": true, "slotRecurrence": { "weekdays": [1, 2, 3, 4, 5] } }
+      ]
+    },
+    {...}
+  ]
+```
+This is an example of a shiftplan which contains a shift name `OneShiftLocation` in `locationId` field.
+The `recurringTimeslots` field is an array which define components of the shift. In this case, this shift has two parts: 
```

Comment:
I guess it is worth mentioning that only "recurringTimeslots" is supported; "timeslots" is not supported (pls verify this statement in the code)